tests/llama_index/test_llama_index_model_export.py

```diff
@@ -3,7 +3,7 @@
 from typing import Any
 from unittest import mock
 
-import llama_index
+import llama_index.core
```

Comment:
will this still work in versions < 0.11.0?

Reply:
Seems it works in the test of 0.10.x versions